doc/classes/Object.xml

```diff
@@ -764,6 +764,23 @@
 				Emits the [signal property_list_changed] signal. This is mainly used to refresh the editor, so that the Inspector and editor plugins are properly updated.
 			</description>
 		</method>
+		<method name="property_can_revert" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="property" type="StringName" />
+			<description>
+				Returns [code]true[/code] if the given [param property] has a custom default value. Use [method property_get_revert] to get the [param property]'s default value.
+				[b]Note:[/b] This method is used by the Inspector dock to display a revert icon. The object must implement [method _property_can_revert] to customize the default value. If [method _property_can_revert] is not implemented, this method returns [code]false[/code].
+				[b]Note:[/b] This method returns consistently, regardless of the current value of the [param property].
```

Comment:
I feel like this note is a bit redundant. It is mostly true, but it's up to the implementation of `_property_can_revert` to decide this.